test/cmd/rsh.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc delete all,deployments --all
+  exit 0
+) &>/dev/null
+
+os::test::junit::declare_suite_start "cmd/rsh"
+os::cmd::expect_success 'oc run test-dc --image=openshift/origin --replicas=2 --command -- /bin/bash -c "sleep infinity"'
+os::cmd::expect_success 'kubectl run test-d --image=openshift/origin --replicas=2 --command -- /bin/bash -c "sleep infinity"'
```

Comment:
Can't you run `oc` with the deployment generator? This may break us the next time the default generator changes upstream.